.github/actions/functional-test-cucumber/action.yml

```
@@ -0,0 +1,19 @@
+name: Functional Tests
+description: Run Functional Tests
+runs:
+  using: "composite"
+  steps:
+    - run: npm install --silent --unsafe-perm
+      shell: bash
+    - run: npm pack
+      shell: bash
+    - run: tar xf kourou-$(node -e 'console.log(require("./package.json").version)').tgz
+      shell: bash
```

Why not just run typescript compile instead of packing and unpacking?

```suggestion
    - run: tsc
      shell: bash
```

---

Because we want to test the generated NPM package and not the source code